### ISSUE TITLE
Add support for page.frameLocator

### ISSUE THREAD

**Issue by tom-miseur** [grafana/k6#4247]
### Feature Description

https://playwright.dev/docs/api/class-framelocator

`page.frameLocator`  appears to cater for waiting on elements within an iframe to appear, something that is necessary in applications featuring iframes.

The current approach involves polling for the frame to appear before then attempting to wait for elements within it.

**Comment by NSnietol** [comment]
Hi @inancgumus could you please let me know in which way I can interact with Iframe as of 2025? I would really appreciate it

**Comment by inancgumus** [comment]
Closing as we have a new issue: #5177.
